### DISCUSSION TITLE
What’s new in Budibase: February 2023  📝↩️👥

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#9952] [Announcements]
As Budibase continues to grow in users and usage we decided to dedicate February to improving the scalability and usability of Budibase.  

---

## Undo/Redo

![Undo/redo image](https://res.cloudinary.com/daog6scxm/image/upload/v1678206223/features/Undo_and_redo_lcgtfj.png)

A small but mighty feature, Undo/Redo has been on our backlog since day 1. With the February release, I’m pleased to announce users can undo/redo actions with the click of a button or using keyboard shortcuts. To undo an action, simply press Ctrl + Z / ⌘ + Z. To redo an undone action, press Ctrl + Shift + Z  / ⌘ + Shift + Z. 

---

## Faster user invite experience

![Faster user invite experience UI](https://res.cloudinary.com/daog6scxm/image/upload/v1678206223/features/New_user_flow_xuidkr.png)

Invite users, assign roles, and configure permissions, right within the data, design, and automation sections. This is a huge improvement and will speed up the user invite flow significantly.

---

## Audit logs

![Audit logs](https://res.cloudinary.com/daog6scxm/image/upload/v1678206223/features/Audit_logs_t1p1hp.png)

Audit logs help you track and analyze your team members’ activity. They’re fully searchable and provide an overview and historical log into nearly every action that can be taken by your team members. Users can also download their audit logs.

Audit logs can be accessed by team members with the Admin role, and are available to customers on the business and enterprise plans. If you're interested in Audit Logs, please [contact sales](https://budibase.com/contact) for a demo.

---

## Enforced SSO 

Enforced SSO allows you to remove email login from the onboarding experience, and ensure all users within your tenant log in using SSO. Enforced SSO is available to customers on the enterprise plan.

---

## Improvements & Fixes

If you're interested in learning more about an improvement/bug fix, copy the issue ID, visit the [Budibase Repo](https://github.com/Budibase/budibase), and paste the ID into Github's search bar.

- Can't add user to a group. #8470
- App locked session contamination #9774
- Cannot Edit Display Column after saving CSV import with no display column chosen. #9929
- Unable to edit formula columns #9927
- Cannot assign Barcode/QR field in form #9523
- Helm install and 'clouseau@127.0.0.1' aborted since it cannot handle ["BIG_CREATION"] error #9487
- Zero Values in Sources/Queries force datatype of column to text resulting in column sorting problems #9393
- Table component displays Invalid Date for dates formatted with handlebars Date helper #8775
- OIDC logos don't appear on auth config screen #7639
- Screen freezes when filtering by date. #9406
- Wildcard S3 domains in NGINX CSP #8042
- Trigger binding not adding on click #8560
- Is in filter no longer directly supports relationship pickers #9015

**Comment by adamjdavies** [comment]
Undo/Redo is a game changer! Thanks for implementing this!! 🙌

**Comment by Nocia-87** [comment]
I greatly appreciate your efforts.
Once the application has been published, is it possible to use it offline?

**Comment by Nocia-87** [comment]
The problem of the columns is not solved on my side
I'm on version 2.4.42
![image](https://user-images.githubusercontent.com/101875102/228940667-ddef314f-81d6-48e5-a093-a5aaf9d592d8.png)
![image](https://user-images.githubusercontent.com/101875102/228940735-422c1ce6-08c8-4b61-97b0-70c75b01d6f9.png)

**Comment by dewdeded** [comment]
Great release, thanks.
